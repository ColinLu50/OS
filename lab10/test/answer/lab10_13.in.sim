3406
906
1008
544
906
463
