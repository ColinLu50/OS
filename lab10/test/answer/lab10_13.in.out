FCFS
428 -> 377 -> 197 -> 246 -> 53 -> 237 -> 386 -> 561 -> 579 -> 274 -> 208 -> 188 -> 516 -> 333 -> 249 -> 230 -> 241 -> 333 -> 577 -> 494 -> 51 -> 124 -> 580
3406
SSTF
428 -> 386 -> 377 -> 333 -> 333 -> 274 -> 249 -> 246 -> 241 -> 237 -> 230 -> 208 -> 197 -> 188 -> 124 -> 53 -> 51 -> 494 -> 516 -> 561 -> 577 -> 579 -> 580
906
SCAN
428 -> 386 -> 377 -> 333 -> 274 -> 249 -> 246 -> 241 -> 237 -> 230 -> 208 -> 197 -> 188 -> 124 -> 53 -> 51 -> 0 -> 494 -> 516 -> 561 -> 577 -> 579 -> 580
1008
C-SCAN
428 -> 386 -> 377 -> 333 -> 274 -> 249 -> 246 -> 241 -> 237 -> 230 -> 208 -> 197 -> 188 -> 124 -> 53 -> 51 -> 0 -> 610 -> 580 -> 579 -> 577 -> 561 -> 516 -> 494
544
LOOK
428 -> 386 -> 377 -> 333 -> 274 -> 249 -> 246 -> 241 -> 237 -> 230 -> 208 -> 197 -> 188 -> 124 -> 53 -> 51 -> 494 -> 516 -> 561 -> 577 -> 579 -> 580
906
C-LOOK
428 -> 386 -> 377 -> 333 -> 274 -> 249 -> 246 -> 241 -> 237 -> 230 -> 208 -> 197 -> 188 -> 124 -> 53 -> 51 -> 580 -> 579 -> 577 -> 561 -> 516 -> 494
463
